.github/workflows/Build and test.yml

```
@@ -33,36 +33,28 @@
-        export PATH="$PWD/avr-toolchain/avr-gcc-13.2.0-x64-linux/bin:$PATH" &&
-        cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}}
+      run: >
+        docker run --rm 
+        -v ${{ github.workspace }}:/Nextag-Embedded-Base
+        -w /Nextag-Embedded-Base/build/${{ matrix.build_type }} avr-gcc-toolchain
+          ninja -j $(nproc)
 
     - name: Test
-      working-directory: ${{github.workspace}}/build
-      run: ctest -C ${{env.BUILD_TYPE}} --output-on-failure
+      run: > 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)